src/assets/icons/deviceIcon.svg

```diff
@@ -0,0 +1,16 @@
+<!-- Copyright (c) Microsoft. All rights reserved. -->
+
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="-20836 -16624 89.406 72">
+  <defs>
```

Comment:
Yup, again... this is from the old project. I didn't invent it. Didn't even look at the contents of the files since they were fromthe old... but I'm happy to fix it. :)